packages/gasket-plugin-docs/lib/get-commands.js

```diff
@@ -30,6 +31,7 @@ module.exports = function getCommands(gasket, { GasketCommand, flags }) {
       }
       docsConfigSet.guides.unshift(...guides);
       await generateIndex(docsConfigSet);
+      await generateLicense(docsConfigSet);
```

Comment:
Hmm, I think you're do more work than is necessary. Couldn't you just add `LICENSE.md` to the docSetup files that it just gets captured along with the readme, etc.?

```diff
module.exports = function docsSetup() {
  return {
    link: 'README.md',
    files: [
      'README.md',
+     'LICENSE.md',
      'docs/**/*'
    ],
    transforms: [
      txGasketPackageLinks,
      txGasketUrlLinks,
      txAbsoluteLinks
      txAbsoluteLinks
    ]
```

Reply:
Yeah I can do that. I didn't know that's how that behaved.